epub34/a11y/index.html

```diff
@@ -1677,31 +1663,31 @@ <h2>Optimized publications</h2>
+					to meet the recommendations in this section to conform to this specification, some jurisdictions
+					require similar distribution practices. <a
 						href="https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882">Directive
 						2019/882</a>, for example, includes similar requirements for digital publications distributed in
 					the European Union.</p>
 			</div>
 
 			<p>The creation of accessible <a href="https://www.w3.org/TR/epub/#dfn-epub-publication">EPUB
 					publications</a> does not guarantee that the publication will be obtainable or consumable by users
-				in an accessible fashion. Depending on how EPUB creators distribute their EPUB publications, other
-				factors will influence their overall accessibility. For example, an accessible interface for locating
-				and obtaining content is an essential part of the distribution process, as is the ability to search and
-				review accessibility metadata.</p>
-
-			<p>While much of the distribution process is outside the control of EPUB creators, so outside the scope of
-				this specification, there are factors an EPUB creator can control. For example, while an EPUB creator
-				typically does not control the accessibility of the digital rights management (DRM) scheme applied to
-				their EPUB publications, they do control what usage rights to apply to their EPUB publications. So even
-				though a DRM scheme may allow an Author to block access to the text of the publication, the EPUB creator
-				needs to take care not to apply such a restriction as it could block the ability for <a>assistive
-					technologies</a> to read the text aloud.</p>
-
-			<p>To minimize the effects of distribution on accessibility, this specification advises EPUB creators adhere
-				to the following distribution practices:</p>
+				in an accessible fashion. Depending on how they are distributed, other factors will influence their
+				overall accessibility. For example, an accessible interface for locating and obtaining content is an
+				essential part of the distribution process, as is the ability to search and review accessibility
+				metadata.</p>
+
+			<p>While many aspects of the distribution process are outside the scope of this specification, there are
+				factors that publishers can control. For example, while a publisher typically does not control the
+				accessibility of the digital rights management (DRM) scheme applied to their EPUB publications, they do
+				control what usage rights to apply to their EPUB publications. So even though a DRM scheme may allow a
+				publisher to block access to the text of the publication, the publisher needs to take care not to apply
+				such a restriction as it could block the ability for <a>assistive technologies</a> to read the text
+				aloud.</p>
+
+			<p>To minimize the effects of distribution on accessibility, this specification advises that publishers
+				adhere to the following distribution practices:</p>
```

Comment:
Ouch! Ok, let me take a shot.
```suggestion
			<p>While many aspects of the distribution process are outside the scope of this specification, there are
				factors that can be controlled. For example, while a publisher typically does not control the
				accessibility of the digital rights management (DRM) scheme applied to their EPUB publications, they do
				control what usage rights to apply to their EPUB publications. So even though a DRM scheme may allow a
				publisher to block access to the text of the publication, that publisher needs to take care not to apply
				such a restriction as it could block the ability for <a>assistive technologies</a> to read the text
				aloud.</p>

			<p>To minimize the effects of distribution on accessibility, this specification advises adherence
			    to the following distribution practices:</p>
```

That at least confines "publisher" to the example, which seems ok.